ister_gui.py

```diff
@@ -644,6 +672,247 @@ def validate_condition(self):
         return True
 
 
+class ChooseAction(ProcessStep):
+    """UI to select installation path (Install, Shell, or Repair)"""
+    def __init__(self):
+        super(ChooseAction, self).__init__()
+        self.choices = {
+            'Install': 'Install          - Install Clear Linux OS',
+            'Repair': 'Repair           - Attempt to repair host OS',
+            'Shell': 'Shell            - Open shell in host OS',
+            'Update Installer': 'Update Installer - Update installer to '
```

Comment:
So swupd update on a fixed size installer image that attempts to be small does not give me much confidence. Also those files are cached in /var and cleanup isn't automated at this point.

I'm just not really convinced adding any code for this is worthwhile and fixes more issues than it brings up.